src/components/Books.jsx

```diff
@@ -14,21 +20,29 @@ const Books = () => {
           <li key={book.id}>
             <span>title: {book.title}</span>
             <span>author: {book.author} </span>
-            <button type="button" id={book.id}>Delete Book {book.id}</button>
+            <button type="button" onClick={() => dispatch(deleteBook(book.id))}>Delete Book</button>
           </li>
         ))}
       </ul>
-      <form action="">
-        <label htmlFor="title">
-          Title
-          <input type="text" name="title" id="title" />
-        </label>
-        <label htmlFor="author">
-          Author
-          <input type="text" name="author" id="author" />
-        </label>
-        <button type="submit">Add Book</button>
-      </form>
+
+      <label htmlFor="title">
+        Title
+        <input type="text" name="title" id="title" value={book.title} onChange={(e) => setBook({ ...book, title: e.target.value })} />
+      </label>
+      <label htmlFor="author">
+        Author
+        <input type="text" name="author" id="author" value={book.author} onChange={(e) => setBook({ ...book, author: e.target.value })} />
+      </label>
+      <button
+        type="submit"
+        onClick={() => {
+          dispatch(addBook(book));
+          setBook({ title: '', author: '' });
+        }}
+      >
+        Add Book
+      </button>
```

Comment:
- [ ] Kindly, wrap these elements in a `form` tag then move the `form` tag in its own component.